src/hooks/wallets/mpc/useMPCWallet.ts

```diff
@@ -99,12 +107,21 @@ export const useMPCWallet = (): MPCWalletHook => {
     if (mpcCoreKit.status === COREKIT_STATUS.LOGGED_IN) {
       await mpcCoreKit.commitChanges()
 
-      await connectWallet(onboard, {
+      const wallets = await connectWallet(onboard, {
         autoSelect: {
           label: ONBOARD_MPC_MODULE_LABEL,
           disableModals: true,
         },
       }).catch((reason) => console.error('Error connecting to MPC module:', reason))
+
+      // If the signer is not in the address book => add it as the user's first name
+      if (wallets && currentChainId && wallets.length > 0) {
+        const signerAddress = ethers.utils.getAddress(wallets[0].accounts[0]?.address)
```

Comment:
`getAddress` throws an error if we pass undefined. Not sure why the types don't reflect it but I think we need to split up the call. We could also reuse our `checksumAddress` wdyt?

```
const address = wallets[0].accounts[0]?.address
if (!address) return
const checksummedAddress = checksumAddress(address)
```